config/initializers/session_store.rb

```diff
@@ -20,4 +20,5 @@
                                        signed: true,
                                        secure: ApplicationConfig["FORCE_SSL_IN_RAILS"] == "true",
                                        same_site: :lax,
-                                       httponly: true
+                                       httponly: true,
+                                       namespace: ENV["REDIS_NAMESPACE"]
```

Comment:
I'm also not sure if this is working since I can see my session Redis key locally and it does not have a namespace.

Reply:
Not sure why but I also noticed this https://github.com/redis-store/redis-store/issues/339 - not all commands are namespaces, same issue redis-namespace has